pkg/apis/config/v1alpha1/resmgr.go

```diff
@@ -41,4 +43,8 @@ type CommonConfig struct {
 	Log log.Config `json:"log,omitempty"`
 	// +optional
 	Instrumentation instrumentation.Config `json:"instrumentation,omitempty"`
+	// +optional
+	RdtClasses rdt.Config `json:"rdt,omitempty"`
+	// +optional
+	BlockIOClasses blockio.Config `json:"blockIO,omitempty"`
```

Comment:
nit/question: just wondering why the field names are *classes as the json name don't have that

```suggestion
	Rdt rdt.Config `json:"rdt,omitempty"`
	// +optional
	BlockIO blockio.Config `json:"blockIO,omitempty"`
```

Reply:
Okay, now that you asked, I just realized that all of these JSON tags are useless and unnecessary. `CommonConfig` is not something we'd ever pass as a type over the wire. It is only used and constructed locally...